dir.props

```diff
@@ -10,11 +10,21 @@
     <!-- true if we have bootstrapped buildtools (usually on an unsupported platform -->
     <_IsBootstrapping Condition="'$(BootstrapBuildToolsDir)' != ''">true</_IsBootstrapping>
 
-    <PortableBuild Condition="'$(OS)' == 'Windows_NT'">true</PortableBuild>
-    <PortableBuild Condition="'$(PortableBuild)' == ''">false</PortableBuild>
-    <UseSystemLibraries Condition="'$(UseSystemLibraries)' == '' AND '$(PortableBuild)' != 'true'">true</UseSystemLibraries>
-    <UseSystemLibunwind Condition="'$(UseSystemLibunwind)' == ''">$(UseSystemLibraries)</UseSystemLibunwind>
     <UseStableVersions Condition="'$(UseStableVersions)' == ''">true</UseStableVersions>
+
+    <!-- new supported portable/nonportable options, default based on old options if set, otherwise just regular defaults -->
+    <BuildPortableRuntime Condition="'(BuildPortableRuntime)' == '' and '$(PortableBuild)' == 'false'">false</BuildPortableRuntime>
```

Comment:
Missing $ in the condition on this line and subsequent lines?

Reply:
Yes, thanks.